src/scripting/mapgen_lua_kernel.cpp

```diff
@@ -249,6 +249,18 @@ mapgen_lua_kernel::mapgen_lua_kernel(const config* vars)
 	lua_pop(L, 1);
 	assert(lua_gettop(L) == 0);
 
+	// Create the paths module
+	cmd_log_ << "Adding paths module...\n";
```

Comment:
This lacks the corresponding removal of `wesnoth.find_path`, which is still there on line 223.

Line 224 is also interesting, that's now bug #5846.